lua/cmp_luasnip/init.lua

```diff
@@ -70,12 +71,18 @@ function source:complete(params, callback)
 		local ft = filetypes[i]
 		if not snip_cache[ft] then
 			-- ft not yet in cache.
-			local ft_items = {}
-			local ft_table = require("luasnip").get_snippets(ft, {
-				type = "snippets"
-			})
-			if ft_table then
-				for j, snip in pairs(ft_table) do
+			local ft_items   = {}
+			local ft_table   = require("luasnip").get_snippets(ft, {type = "snippets"})
+			local auto_table = require('luasnip').get_snippets(ft, {type="autosnippets"})
```

Comment:
Oh, there's no need to query autosnippets if they are disabled, moving this into the `if` should be fine right?

Reply:
Yes right :+1:
Done